syz-fuzzer/fuzzer.go

```diff
@@ -136,6 +136,12 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
+
+	if fd, err := syscall.Open("/dev/net/tun", syscall.O_RDWR, 0); err == nil {
```

Comment:
This duplicates logic in host package. And this actually does it in a different way: here we open, there we stat and check uid. This is a recipe for future bugs. Also you don't account for enable_syscalls.
Instead check calls[sys.CallMap["syz_emit_ethernet"]].